Param value not trimmed anymore

### Current Behavior

```<Param name="STN" xpathExpression="/record/STN" />``` now gives:

(NumberFormatException) For input string: "  235"

This can be fixed with:

```<Param name="STN" xpathExpression="translate(/record/STN, ' ', '')" />```

### Expected Behavior

No error

### Environment Information

FF! 10.2.0-20260522.042328: Frank2Example3
Running on xxx using Apache Tomcat/10.1.55
Java Version: OpenJDK Runtime Environment (25.0.3+9-LTS)
Heap size: 335.0 MiB, total JVM memory: 444.0 MiB
Free memory: 109.0 MiB, max memory: 1.0 GiB


### Steps To Reproduce

Ask me when needed

### Configuration

```xml
Ask me when needed
```

### Input

_No response_

### What database are you using?

_No response_

### What browsers are you seeing the problem on?

_No response_

### Relevant Log Output

```console

```

### Anything else?

_No response_

Stacktrace with 10.2.0-20260609.042330:

```
<errorMessage timestamp="Wed Jun 10 12:54:27 CEST 2026" originator="IAF 10.2.0-20260609.042330" message="ForEachChildElementPipe [Insert into table daggegevens] msgId [testmessage-0a000066-15379153_19eb129e969_-7fc2]: error during pipeline processing: Pipe [Insert into table daggegevens] caught exception: (SaxException) Could not process list of elements using xpath [/csv/record]: (TransformerException): (SaxException): exception calling JavaListener [Store record]: Pipe [Store in database] caught exception: got exception sending message: Could not set parameter [STN] with type [STRING] at position 0, exception: (SQLException) Could not convert [  235] for parameter [1]: (NumberFormatException) For input string: &quot;  235&quot;">
	<location class="org.frankframework.pipes.ForEachChildElementPipe" name="Insert into table daggegevens"/>
	<details>org.frankframework.core.ListenerException: Pipe [Insert into table daggegevens] caught exception: (SaxException) Could not process list of elements using xpath [/csv/record]: (TransformerException): (SaxException): exception calling JavaListener [Store record]: Pipe [Store in database] caught exception: got exception sending message: Could not set parameter [STN] with type [STRING] at position 0, exception: (SQLException) Could not convert [  235] for parameter [1]: (NumberFormatException) For input string: "  235"
	at org.frankframework.core.Adapter.processMessageWithExceptions(Adapter.java:843)
	at org.frankframework.core.Adapter.processMessageDirect(Adapter.java:768)
	at org.frankframework.management.bus.endpoints.TestPipeline.processMessage(TestPipeline.java:142)
	at org.frankframework.management.bus.endpoints.TestPipeline.runTestPipeline(TestPipeline.java:113)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:158)
	at org.springframework.security.authorization.method.AuthorizationManagerBeforeMethodInterceptor.proceed(AuthorizationManagerBeforeMethodInterceptor.java:271)
	at org.springframework.security.authorization.method.AuthorizationManagerBeforeMethodInterceptor.attemptAuthorization(AuthorizationManagerBeforeMethodInterceptor.java:266)
	at org.springframework.security.authorization.method.AuthorizationManagerBeforeMethodInterceptor.invoke(AuthorizationManagerBeforeMethodInterceptor.java:197)
	at org.springframework.security.config.annotation.method.configuration.DeferringMethodInterceptor.invoke(DeferringMethodInterceptor.java:44)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.security.authorization.method.AuthorizationManagerBeforeMethodInterceptor.proceed(AuthorizationManagerBeforeMethodInterceptor.java:271)
	at org.springframework.security.authorization.method.AuthorizationManagerBeforeMethodInterceptor.attemptAuthorization(AuthorizationManagerBeforeMethodInterceptor.java:266)
	at org.springframework.security.authorization.method.AuthorizationManagerBeforeMethodInterceptor.invoke(AuthorizationManagerBeforeMethodInterceptor.java:197)
	at org.springframework.security.config.annotation.method.configuration.DeferringMethodInterceptor.invoke(DeferringMethodInterceptor.java:44)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:719)
	at org.frankframework.management.bus.endpoints.TestPipeline$$SpringCGLIB$$0.runTestPipeline(&lt;generated&gt;)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.messaging.handler.invocation.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:168)
	at org.springframework.integration.handler.support.IntegrationInvocableHandlerMethod.doInvoke(IntegrationInvocableHandlerMethod.java:47)
	at org.springframework.messaging.handler.invocation.InvocableHandlerMethod.invoke(InvocableHandlerMethod.java:119)
	at org.springframework.integration.handler.support.MessagingMethodInvokerHelper$HandlerMethod.invoke(MessagingMethodInvokerHelper.java:1111)
	at org.springframework.integration.handler.support.MessagingMethodInvokerHelper.invokeHandlerMethod(MessagingMethodInvokerHelper.java:576)
	at org.springframework.integration.handler.support.MessagingMethodInvokerHelper.processInternal(MessagingMethodInvokerHelper.java:489)
	at org.springframework.integration.handler.support.MessagingMethodInvokerHelper.process(MessagingMethodInvokerHelper.java:364)
	at org.springframework.integration.handler.MethodInvokingMessageProcessor.processMessage(MethodInvokingMessageProcessor.java:116)
	at org.springframework.integration.handler.ServiceActivatingHandler.handleRequestMessage(ServiceActivatingHandler.java:97)
	at org.springframework.integration.handler.AbstractReplyProducingMessageHandler.handleMessageInternal(AbstractReplyProducingMessageHandler.java:146)
	at org.springframework.integration.handler.AbstractMessageHandler.doHandleMessage(AbstractMessageHandler.java:102)
	at org.springframework.integration.handler.AbstractMessageHandler.handleMessage(AbstractMessageHandler.java:70)
	at org.springframework.integration.handler.MessageHandlerChain.lambda$configureChain$0(MessageHandlerChain.java:133)
	at org.springframework.messaging.core.GenericMessagingTemplate.doSend(GenericMessagingTemplate.java:187)
	at org.springframework.messaging.core.GenericMessagingTemplate.doSend(GenericMessagingTemplate.java:166)
	at org.springframework.messaging.core.GenericMessagingTemplate.doSend(GenericMessagingTemplate.java:47)
	at org.springframework.messaging.core.AbstractMessageSendingTemplate.send(AbstractMessageSendingTemplate.java:108)
	at org.springframework.integration.handler.AbstractMessageProducingHandler.sendOutput(AbstractMessageProducingHandler.java:556)
	at org.springframework.integration.handler.AbstractMessageProducingHandler.doProduceOutput(AbstractMessageProducingHandler.java:370)
	at org.springframework.integration.handler.AbstractMessageProducingHandler.produceOutput(AbstractMessageProducingHandler.java:289)
	at org.springframework.integration.handler.AbstractMessageProducingHandler.sendOutputs(AbstractMessageProducingHandler.java:253)
	at org.springframework.integration.handler.AbstractReplyProducingMessageHandler.handleMessageInternal(AbstractReplyProducingMessageHandler.java:152)
	at org.springframework.integration.handler.AbstractMessageHandler.doHandleMessage(AbstractMessageHandler.java:102)
	at org.springframework.integration.handler.AbstractMessageHandler.handleMessage(AbstractMessageHandler.java:70)
	at org.springframework.integration.handler.MessageHandlerChain.handleMessageInternal(MessageHandlerChain.java:161)
	at org.springframework.integration.handler.AbstractMessageHandler.doHandleMessage(AbstractMessageHandler.java:102)
	at org.springframework.integration.handler.AbstractMessageHandler.handleMessage(AbstractMessageHandler.java:70)
	at org.springframework.integration.dispatcher.BroadcastingDispatcher.invokeHandler(BroadcastingDispatcher.java:239)
	at org.springframework.integration.dispatcher.BroadcastingDispatcher.dispatch(BroadcastingDispatcher.java:196)
	at org.springframework.integration.channel.AbstractSubscribableChannel.doSend(AbstractSubscribableChannel.java:72)
	at org.springframework.integration.channel.AbstractMessageChannel.sendInternal(AbstractMessageChannel.java:438)
	at org.springframework.integration.channel.AbstractMessageChannel.send(AbstractMessageChannel.java:341)
	at org.springframework.messaging.core.GenericMessagingTemplate.doSend(GenericMessagingTemplate.java:187)
	at org.springframework.messaging.core.GenericMessagingTemplate.doSendAndReceive(GenericMessagingTemplate.java:231)
	at org.springframework.messaging.core.GenericMessagingTemplate.doSendAndReceive(GenericMessagingTemplate.java:47)
	at org.springframework.messaging.core.AbstractMessagingTemplate.sendAndReceive(AbstractMessagingTemplate.java:47)
	at org.springframework.integration.core.MessagingTemplate.sendAndReceive(MessagingTemplate.java:111)
	at org.springframework.integration.gateway.MessagingGatewaySupport.doSendAndReceive(MessagingGatewaySupport.java:681)
	at org.springframework.integration.gateway.MessagingGatewaySupport.sendAndReceive(MessagingGatewaySupport.java:609)
	at org.springframework.integration.gateway.MessagingGatewaySupport.sendAndReceiveMessage(MessagingGatewaySupport.java:583)
	at org.frankframework.management.bus.LocalGateway.sendSyncMessage(LocalGateway.java:50)
	at org.frankframework.console.controllers.FrankApiService.sendSyncMessage(FrankApiService.java:80)
	at org.frankframework.console.controllers.TestPipeline.testPipeLine(TestPipeline.java:104)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:158)
	at org.springframework.security.authorization.method.AuthorizationManagerBeforeMethodInterceptor.proceed(AuthorizationManagerBeforeMethodInterceptor.java:271)
	at org.springframework.security.authorization.method.AuthorizationManagerBeforeMethodInterceptor.attemptAuthorization(AuthorizationManagerBeforeMethodInterceptor.java:266)
	at org.springframework.security.authorization.method.AuthorizationManagerBeforeMethodInterceptor.invoke(AuthorizationManagerBeforeMethodInterceptor.java:197)
	at org.springframework.security.config.annotation.method.configuration.DeferringMethodInterceptor.invoke(DeferringMethodInterceptor.java:44)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:719)
	at org.frankframework.console.controllers.TestPipeline$$SpringCGLIB$$0.testPipeLine(&lt;generated&gt;)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.web.method.support.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:252)
	at org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest(InvocableHandlerMethod.java:184)
	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:117)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:934)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:853)
	at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:86)
	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:963)
	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:866)
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1000)
	at org.springframework.web.servlet.FrameworkServlet.doPost(FrameworkServlet.java:903)
	at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:590)
	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:874)
	at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:658)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:138)
	at org.frankframework.console.filters.SecurityLogFilter.doFilterInternal(SecurityLogFilter.java:46)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:138)
	at org.frankframework.console.filters.CsrfCookieFilter.doFilterInternal(CsrfCookieFilter.java:48)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:138)
	at org.springframework.web.filter.ShallowEtagHeaderFilter.doFilterInternal(ShallowEtagHeaderFilter.java:107)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:138)
	at org.frankframework.console.filters.CacheControlFilter.doFilter(CacheControlFilter.java:52)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:138)
	at org.frankframework.console.filters.CorsFilter.doFilter(CorsFilter.java:114)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:138)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:51)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:138)
	at org.springframework.web.filter.CompositeFilter$VirtualFilterChain.doFilter(CompositeFilter.java:108)
	at org.springframework.security.web.FilterChainProxy.lambda$doFilterInternal$3(FilterChainProxy.java:235)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:376)
	at org.springframework.security.web.access.intercept.AuthorizationFilter.doFilter(AuthorizationFilter.java:101)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:126)
	at org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:120)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.session.SessionManagementFilter.doFilter(SessionManagementFilter.java:92)
	at org.springframework.security.web.session.SessionManagementFilter.doFilter(SessionManagementFilter.java:86)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.authentication.AnonymousAuthenticationFilter.doFilter(AnonymousAuthenticationFilter.java:100)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter.doFilter(SecurityContextHolderAwareRequestFilter.java:181)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.savedrequest.RequestCacheAwareFilter.doFilter(RequestCacheAwareFilter.java:63)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:110)
	at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:96)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:110)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.context.SecurityContextHolderFilter.doFilter(SecurityContextHolderFilter.java:75)
	at org.springframework.security.web.context.SecurityContextHolderFilter.doFilter(SecurityContextHolderFilter.java:69)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:101)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:110)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:110)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:237)
	at org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:190)
	at org.springframework.web.filter.CompositeFilter$VirtualFilterChain.doFilter(CompositeFilter.java:113)
	at org.springframework.web.filter.ServletRequestPathFilter.doFilter(ServletRequestPathFilter.java:52)
	at org.springframework.web.filter.CompositeFilter$VirtualFilterChain.doFilter(CompositeFilter.java:113)
	at org.springframework.web.filter.CompositeFilter.doFilter(CompositeFilter.java:74)
	at org.springframework.security.config.annotation.web.configuration.WebSecurityConfiguration$CompositeFilterChainProxy.doFilter(WebSecurityConfiguration.java:317)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:138)
	at org.springframework.web.filter.CompositeFilter$VirtualFilterChain.doFilter(CompositeFilter.java:108)
	at org.springframework.security.web.FilterChainProxy.lambda$doFilterInternal$3(FilterChainProxy.java:235)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:376)
	at org.springframework.security.web.access.intercept.AuthorizationFilter.doFilter(AuthorizationFilter.java:101)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:126)
	at org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:120)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.session.SessionManagementFilter.doFilter(SessionManagementFilter.java:132)
	at org.springframework.security.web.session.SessionManagementFilter.doFilter(SessionManagementFilter.java:86)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.authentication.AnonymousAuthenticationFilter.doFilter(AnonymousAuthenticationFilter.java:100)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter.doFilter(SecurityContextHolderAwareRequestFilter.java:181)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.savedrequest.RequestCacheAwareFilter.doFilter(RequestCacheAwareFilter.java:63)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:110)
	at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:96)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.header.HeaderWriterFilter.doHeadersAfter(HeaderWriterFilter.java:90)
	at org.springframework.security.web.header.HeaderWriterFilter.doFilterInternal(HeaderWriterFilter.java:75)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.context.SecurityContextHolderFilter.doFilter(SecurityContextHolderFilter.java:82)
	at org.springframework.security.web.context.SecurityContextHolderFilter.doFilter(SecurityContextHolderFilter.java:69)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:101)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter.doFilterInternal(WebAsyncManagerIntegrationFilter.java:62)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.session.DisableEncodeUrlFilter.doFilterInternal(DisableEncodeUrlFilter.java:42)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:237)
	at org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:195)
	at org.springframework.web.filter.CompositeFilter$VirtualFilterChain.doFilter(CompositeFilter.java:113)
	at org.springframework.web.filter.ServletRequestPathFilter.doFilter(ServletRequestPathFilter.java:52)
	at org.springframework.web.filter.CompositeFilter$VirtualFilterChain.doFilter(CompositeFilter.java:113)
	at org.springframework.web.filter.CompositeFilter.doFilter(CompositeFilter.java:74)
	at org.springframework.security.config.annotation.web.configuration.WebSecurityConfiguration$CompositeFilterChainProxy.doFilter(WebSecurityConfiguration.java:317)
	at org.springframework.web.filter.DelegatingFilterProxy.invokeDelegate(DelegatingFilterProxy.java:355)
	at org.springframework.web.filter.DelegatingFilterProxy.doFilter(DelegatingFilterProxy.java:272)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:138)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:165)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:88)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:609)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:113)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:83)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:782)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:72)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:342)
	at org.apache.coyote.http2.StreamProcessor.service(StreamProcessor.java:476)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:63)
	at org.apache.coyote.http2.StreamProcessor.process(StreamProcessor.java:102)
	at org.apache.coyote.http2.StreamRunnable.run(StreamRunnable.java:35)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:973)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:491)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
	at java.base/java.lang.Thread.run(Thread.java:1474)
Caused by: org.frankframework.core.PipeRunException: Pipe [Insert into table daggegevens] caught exception: (SaxException) Could not process list of elements using xpath [/csv/record]: (TransformerException): (SaxException): exception calling JavaListener [Store record]: Pipe [Store in database] caught exception: got exception sending message: Could not set parameter [STN] with type [STRING] at position 0, exception: (SQLException) Could not convert [  235] for parameter [1]: (NumberFormatException) For input string: "  235"
	at org.frankframework.pipes.MessageSendingPipe.doPipe(MessageSendingPipe.java:375)
	at org.frankframework.processors.CorePipeProcessor.processPipe(CorePipeProcessor.java:36)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.TrackPreviousPipeInMetadataProcessor.processPipe(TrackPreviousPipeInMetadataProcessor.java:42)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.CheckMessageSizePipeProcessor.processPipe(CheckMessageSizePipeProcessor.java:38)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.LockerPipeProcessor.processPipe(LockerPipeProcessor.java:40)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.TransactionAttributePipeProcessor.executePipeProcess(TransactionAttributePipeProcessor.java:96)
	at org.frankframework.processors.TransactionAttributePipeProcessor.processPipe(TransactionAttributePipeProcessor.java:66)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.LimitingParallelExecutionPipeProcessor.processPipe(LimitingParallelExecutionPipeProcessor.java:50)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.LimitingParallelExecutionPipeProcessor.processPipe(LimitingParallelExecutionPipeProcessor.java:77)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:158)
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:94)
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugPipeGetInputFrom(IbisDebuggerAdvice.java:187)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:648)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:630)
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:70)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:719)
	at org.frankframework.processors.LimitingParallelExecutionPipeProcessor$$SpringCGLIB$$0.processPipe(&lt;generated&gt;)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.ExceptionHandlingPipeProcessor.processPipe(ExceptionHandlingPipeProcessor.java:40)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.InputOutputPipeProcessor.processPipe(InputOutputPipeProcessor.java:97)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.InputOutputPipeProcessor.processPipe(InputOutputPipeProcessor.java:240)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:158)
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:94)
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugPipeInputOutputAbort(IbisDebuggerAdvice.java:159)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:648)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:630)
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:70)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:719)
	at org.frankframework.processors.InputOutputPipeProcessor$$SpringCGLIB$$0.processPipe(&lt;generated&gt;)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.MonitoringPipeProcessor.processPipe(MonitoringPipeProcessor.java:40)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.LogPipeProcessor.processPipe(LogPipeProcessor.java:56)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.CorePipeLineProcessor.runToExit(CorePipeLineProcessor.java:222)
	at org.frankframework.processors.CorePipeLineProcessor.processPipeLine(CorePipeLineProcessor.java:64)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:158)
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:94)
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugPipeLineInputOutputAbort(IbisDebuggerAdvice.java:124)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:648)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:630)
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:70)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:719)
	at org.frankframework.processors.CorePipeLineProcessor$$SpringCGLIB$$0.processPipeLine(&lt;generated&gt;)
	at org.frankframework.processors.LockerPipeLineProcessor.processPipeLine(LockerPipeLineProcessor.java:50)
	at org.frankframework.processors.TransactionAttributePipeLineProcessor.processPipeLine(TransactionAttributePipeLineProcessor.java:51)
	at org.frankframework.processors.LimitingParallelExecutionPipeLineProcessor.processPipeLine(LimitingParallelExecutionPipeLineProcessor.java:45)
	at org.frankframework.processors.CachePipeLineProcessor.processPipeLine(CachePipeLineProcessor.java:43)
	at org.frankframework.processors.InputOutputPipeLineProcessor.processPipeLine(InputOutputPipeLineProcessor.java:48)
	at org.frankframework.core.PipeLine.process(PipeLine.java:441)
	at org.frankframework.core.Adapter.processMessageWithExceptions(Adapter.java:840)
	... 217 more
Caused by: org.frankframework.core.SenderException: (SaxException) Could not process list of elements using xpath [/csv/record]: (TransformerException): (SaxException): exception calling JavaListener [Store record]: Pipe [Store in database] caught exception: got exception sending message: Could not set parameter [STN] with type [STRING] at position 0, exception: (SQLException) Could not convert [  235] for parameter [1]: (NumberFormatException) For input string: "  235"
	at org.frankframework.pipes.ForEachChildElementPipe.iterateOverInput(ForEachChildElementPipe.java:402)
	at org.frankframework.pipes.IteratingPipe.sendMessage(IteratingPipe.java:477)
	at org.frankframework.pipes.MessageSendingPipe.sendMessageWithRetries(MessageSendingPipe.java:414)
	at org.frankframework.pipes.MessageSendingPipe.doPipe(MessageSendingPipe.java:347)
	... 308 more
Caused by: org.frankframework.xml.SaxException: Could not process list of elements using xpath [/csv/record]
	at org.frankframework.pipes.ForEachChildElementPipe$1.handleException(ForEachChildElementPipe.java:353)
	at org.frankframework.xml.AbstractExceptionCatchingFilter.characters(AbstractExceptionCatchingFilter.java:65)
	at org.apache.xerces.parsers.AbstractSAXParser.characters(Unknown Source)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanContent(Unknown Source)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl$FragmentContentDispatcher.dispatch(Unknown Source)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source)
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source)
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source)
	at org.apache.xerces.parsers.XMLParser.parse(Unknown Source)
	at org.apache.xerces.parsers.AbstractSAXParser.parse(Unknown Source)
	at org.apache.xerces.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source)
	at org.frankframework.util.XmlUtils.parseXml(XmlUtils.java:259)
	at org.frankframework.util.XmlUtils.parseXml(XmlUtils.java:246)
	at org.frankframework.pipes.ForEachChildElementPipe.iterateOverInput(ForEachChildElementPipe.java:392)
	... 311 more
Caused by: javax.xml.transform.TransformerException: org.frankframework.xml.SaxException: exception calling JavaListener [Store record]: Pipe [Store in database] caught exception: got exception sending message: Could not set parameter [STN] with type [STRING] at position 0, exception: (SQLException) Could not convert [  235] for parameter [1]: (NumberFormatException) For input string: "  235"
	at org.apache.xalan.templates.ElemCopyOf.execute(ElemCopyOf.java:212)
	at org.apache.xalan.transformer.TransformerImpl.executeChildTemplates(TransformerImpl.java:2402)
	at org.apache.xalan.templates.ElemElement.constructNode(ElemElement.java:341)
	at org.apache.xalan.templates.ElemElement.execute(ElemElement.java:290)
	at org.apache.xalan.transformer.TransformerImpl.executeChildTemplates(TransformerImpl.java:2402)
	at org.apache.xalan.transformer.TransformerImpl.applyTemplateToNode(TransformerImpl.java:2272)
	at org.apache.xalan.transformer.TransformerImpl.transformNode(TransformerImpl.java:1358)
	at org.apache.xalan.transformer.TransformerImpl.run(TransformerImpl.java:3449)
	... 1 more
Caused by: org.frankframework.xml.SaxException: exception calling JavaListener [Store record]: Pipe [Store in database] caught exception: got exception sending message: Could not set parameter [STN] with type [STRING] at position 0, exception: (SQLException) Could not convert [  235] for parameter [1]: (NumberFormatException) For input string: "  235"
	at org.frankframework.pipes.ForEachChildElementPipe$ItemCallbackCallingHandler.endNode(ForEachChildElementPipe.java:212)
	at org.frankframework.xml.NodeSetFilter.endElement(NodeSetFilter.java:163)
	at org.frankframework.pipes.ForEachChildElementPipe$ItemCallbackCallingHandler.endElement(ForEachChildElementPipe.java:231)
	at java.xml/org.xml.sax.helpers.XMLFilterImpl.endElement(XMLFilterImpl.java:558)
	at org.frankframework.documentbuilder.xml.XmlTee.endElement(XmlTee.java:64)
	at java.xml/org.xml.sax.helpers.XMLFilterImpl.endElement(XMLFilterImpl.java:558)
	at org.frankframework.xml.NamespaceRemovingFilter.endElement(NamespaceRemovingFilter.java:35)
	at java.xml/org.xml.sax.helpers.XMLFilterImpl.endElement(XMLFilterImpl.java:558)
	at org.frankframework.xml.AbstractExceptionCatchingFilter.endElement(AbstractExceptionCatchingFilter.java:72)
	at org.apache.xml.serializer.ToXMLSAXHandler.endElement(ToXMLSAXHandler.java:263)
	at org.apache.xml.dtm.ref.DTMTreeWalker.endNode(DTMTreeWalker.java:375)
	at org.apache.xalan.transformer.TreeWalker2Result.endNode(TreeWalker2Result.java:87)
	at org.apache.xml.dtm.ref.DTMTreeWalker.traverse(DTMTreeWalker.java:134)
	at org.apache.xalan.transformer.TreeWalker2Result.traverse(TreeWalker2Result.java:74)
	at org.apache.xalan.templates.ElemCopyOf.execute(ElemCopyOf.java:186)
	... 8 more
Caused by: org.frankframework.core.SenderException: exception calling JavaListener [Store record]: Pipe [Store in database] caught exception: got exception sending message: Could not set parameter [STN] with type [STRING] at position 0, exception: (SQLException) Could not convert [  235] for parameter [1]: (NumberFormatException) For input string: "  235"
	at org.frankframework.senders.IbisLocalSender.sendMessage(IbisLocalSender.java:284)
	at org.frankframework.core.ISender.sendMessageOrThrow(ISender.java:72)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:158)
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:94)
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugSenderInputOutputAbort(IbisDebuggerAdvice.java:231)
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugSenderSendMessageOrThrow(IbisDebuggerAdvice.java:284)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:648)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:630)
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:70)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:719)
	at org.frankframework.senders.IbisLocalSender$$SpringCGLIB$$0.sendMessageOrThrow(&lt;generated&gt;)
	at org.frankframework.pipes.IteratingPipe$ItemCallback.handleItem(IteratingPipe.java:329)
	at org.frankframework.pipes.StringIteratorPipe$1.handleItem(StringIteratorPipe.java:126)
	at org.frankframework.pipes.StringIteratorPipe$1.handleItem(StringIteratorPipe.java:68)
	at org.frankframework.pipes.ForEachChildElementPipe$ItemCallbackCallingHandler.endNode(ForEachChildElementPipe.java:208)
	... 22 more
Caused by: org.frankframework.core.ListenerException: Pipe [Store in database] caught exception: got exception sending message: Could not set parameter [STN] with type [STRING] at position 0, exception: (SQLException) Could not convert [  235] for parameter [1]: (NumberFormatException) For input string: "  235"
	at org.frankframework.core.Adapter.processMessageWithExceptions(Adapter.java:843)
	at org.frankframework.receivers.Receiver.processMessageInAdapter(Receiver.java:1271)
	at org.frankframework.receivers.Receiver.processRequest(Receiver.java:1047)
	at org.frankframework.receivers.JavaListener.processRequest(JavaListener.java:181)
	at org.frankframework.receivers.JavaListener.processRequest(JavaListener.java:170)
	at org.frankframework.senders.IbisLocalSender.sendMessage(IbisLocalSender.java:277)
	... 45 more
Caused by: org.frankframework.core.PipeRunException: Pipe [Store in database] caught exception: got exception sending message: Could not set parameter [STN] with type [STRING] at position 0, exception: (SQLException) Could not convert [  235] for parameter [1]: (NumberFormatException) For input string: "  235"
	at org.frankframework.pipes.MessageSendingPipe.doPipe(MessageSendingPipe.java:375)
	at org.frankframework.processors.CorePipeProcessor.processPipe(CorePipeProcessor.java:36)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.TrackPreviousPipeInMetadataProcessor.processPipe(TrackPreviousPipeInMetadataProcessor.java:42)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.CheckMessageSizePipeProcessor.processPipe(CheckMessageSizePipeProcessor.java:38)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.LockerPipeProcessor.processPipe(LockerPipeProcessor.java:40)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.TransactionAttributePipeProcessor.executePipeProcess(TransactionAttributePipeProcessor.java:96)
	at org.frankframework.processors.TransactionAttributePipeProcessor.processPipe(TransactionAttributePipeProcessor.java:66)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.LimitingParallelExecutionPipeProcessor.processPipe(LimitingParallelExecutionPipeProcessor.java:50)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.LimitingParallelExecutionPipeProcessor.processPipe(LimitingParallelExecutionPipeProcessor.java:77)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:158)
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:94)
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugPipeGetInputFrom(IbisDebuggerAdvice.java:187)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:648)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:630)
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:70)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:719)
	at org.frankframework.processors.LimitingParallelExecutionPipeProcessor$$SpringCGLIB$$0.processPipe(&lt;generated&gt;)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.ExceptionHandlingPipeProcessor.processPipe(ExceptionHandlingPipeProcessor.java:40)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.InputOutputPipeProcessor.processPipe(InputOutputPipeProcessor.java:97)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.InputOutputPipeProcessor.processPipe(InputOutputPipeProcessor.java:240)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:158)
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:94)
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugPipeInputOutputAbort(IbisDebuggerAdvice.java:159)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:648)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:630)
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:70)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:719)
	at org.frankframework.processors.InputOutputPipeProcessor$$SpringCGLIB$$0.processPipe(&lt;generated&gt;)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.MonitoringPipeProcessor.processPipe(MonitoringPipeProcessor.java:40)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.LogPipeProcessor.processPipe(LogPipeProcessor.java:56)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.CorePipeLineProcessor.runToExit(CorePipeLineProcessor.java:222)
	at org.frankframework.processors.CorePipeLineProcessor.processPipeLine(CorePipeLineProcessor.java:64)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:158)
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:94)
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugPipeLineInputOutputAbort(IbisDebuggerAdvice.java:124)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:648)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:630)
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:70)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:719)
	at org.frankframework.processors.CorePipeLineProcessor$$SpringCGLIB$$0.processPipeLine(&lt;generated&gt;)
	at org.frankframework.processors.LockerPipeLineProcessor.processPipeLine(LockerPipeLineProcessor.java:62)
	at org.frankframework.processors.TransactionAttributePipeLineProcessor.processPipeLine(TransactionAttributePipeLineProcessor.java:51)
	at org.frankframework.processors.LimitingParallelExecutionPipeLineProcessor.processPipeLine(LimitingParallelExecutionPipeLineProcessor.java:45)
	at org.frankframework.processors.CachePipeLineProcessor.processPipeLine(CachePipeLineProcessor.java:43)
	at org.frankframework.processors.InputOutputPipeLineProcessor.processPipeLine(InputOutputPipeLineProcessor.java:48)
	at org.frankframework.core.PipeLine.process(PipeLine.java:441)
	at org.frankframework.core.Adapter.processMessageWithExceptions(Adapter.java:840)
	... 50 more
Caused by: org.frankframework.core.SenderException: got exception sending message: Could not set parameter [STN] with type [STRING] at position 0, exception: (SQLException) Could not convert [  235] for parameter [1]: (NumberFormatException) For input string: "  235"
	at org.frankframework.jdbc.AbstractJdbcQuerySender.executeStatementSet(AbstractJdbcQuerySender.java:356)
	at org.frankframework.jdbc.FixedQuerySender.sendMessage(FixedQuerySender.java:131)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:158)
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:94)
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugSenderInputOutputAbort(IbisDebuggerAdvice.java:231)
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugBlockEnabledSenderInputOutputAbort(IbisDebuggerAdvice.java:292)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:648)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:630)
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:70)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:719)
	at org.frankframework.jdbc.FixedQuerySender$$SpringCGLIB$$0.sendMessage(&lt;generated&gt;)
	at org.frankframework.jdbc.AbstractJdbcSender.sendMessage(AbstractJdbcSender.java:132)
	at org.frankframework.pipes.MessageSendingPipe.sendMessage(MessageSendingPipe.java:621)
	at org.frankframework.pipes.MessageSendingPipe.sendMessageWithRetries(MessageSendingPipe.java:414)
	at org.frankframework.pipes.MessageSendingPipe.doPipe(MessageSendingPipe.java:347)
	... 141 more
Caused by: org.frankframework.dbms.JdbcException: Could not set parameter [STN] with type [STRING] at position 0, exception: (SQLException) Could not convert [  235] for parameter [1]: (NumberFormatException) For input string: "  235"
	at org.frankframework.util.JdbcUtil.applyParameters(JdbcUtil.java:491)
	at org.frankframework.util.JdbcUtil.applyParameters(JdbcUtil.java:474)
	at org.frankframework.jdbc.AbstractJdbcQuerySender.executeStatementSet(AbstractJdbcQuerySender.java:321)
	... 164 more
Caused by: java.sql.SQLException: Could not convert [  235] for parameter [1]
	at org.frankframework.util.JdbcUtil.setParameter(JdbcUtil.java:657)
	at org.frankframework.util.JdbcUtil.setParameter(JdbcUtil.java:605)
	at org.frankframework.util.JdbcUtil.applyParameter(JdbcUtil.java:599)
	at org.frankframework.util.JdbcUtil.applyParameters(JdbcUtil.java:487)
	... 166 more
Caused by: java.lang.NumberFormatException: For input string: "  235"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:565)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at org.frankframework.util.JdbcUtil.setParameter(JdbcUtil.java:630)
	... 169 more
</details>
	<originalMessage messageId="testmessage-0a000066-15379153_19eb129e969_-7fc2" receivedTime="Wed Jun 10 12:54:15 CEST 2026" nil="true"/>
</errorMessage>
```

Yes, I tried different param too. Just tried again:

```
<errorMessage timestamp="Wed Jun 10 12:58:20 CEST 2026" originator="IAF 10.2.0-20260609.042330" message="ForEachChildElementPipe [Insert into table daggegevens] msgId [testmessage-0a000066-15379153_19eb129e969_-7fb0]: error during pipeline processing: Pipe [Insert into table daggegevens] caught exception: (SaxException) Could not process list of elements using xpath [/csv/record]: (TransformerException): (SaxException): exception calling JavaListener [Store record]: Pipe [Store in database] caught exception: got exception sending message: Parameter [STN] could not parse result [Message[8887d5b:String]: [  235]] to integer: (NumberFormatException) For input string: &quot;  235&quot;">
	<location class="org.frankframework.pipes.ForEachChildElementPipe" name="Insert into table daggegevens"/>
	<details>org.frankframework.core.ListenerException: Pipe [Insert into table daggegevens] caught exception: (SaxException) Could not process list of elements using xpath [/csv/record]: (TransformerException): (SaxException): exception calling JavaListener [Store record]: Pipe [Store in database] caught exception: got exception sending message: Parameter [STN] could not parse result [Message[8887d5b:String]: [  235]] to integer: (NumberFormatException) For input string: "  235"
	at org.frankframework.core.Adapter.processMessageWithExceptions(Adapter.java:843)
	at org.frankframework.core.Adapter.processMessageDirect(Adapter.java:768)
	at org.frankframework.management.bus.endpoints.TestPipeline.processMessage(TestPipeline.java:142)
	at org.frankframework.management.bus.endpoints.TestPipeline.runTestPipeline(TestPipeline.java:113)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:158)
	at org.springframework.security.authorization.method.AuthorizationManagerBeforeMethodInterceptor.proceed(AuthorizationManagerBeforeMethodInterceptor.java:271)
	at org.springframework.security.authorization.method.AuthorizationManagerBeforeMethodInterceptor.attemptAuthorization(AuthorizationManagerBeforeMethodInterceptor.java:266)
	at org.springframework.security.authorization.method.AuthorizationManagerBeforeMethodInterceptor.invoke(AuthorizationManagerBeforeMethodInterceptor.java:197)
	at org.springframework.security.config.annotation.method.configuration.DeferringMethodInterceptor.invoke(DeferringMethodInterceptor.java:44)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.security.authorization.method.AuthorizationManagerBeforeMethodInterceptor.proceed(AuthorizationManagerBeforeMethodInterceptor.java:271)
	at org.springframework.security.authorization.method.AuthorizationManagerBeforeMethodInterceptor.attemptAuthorization(AuthorizationManagerBeforeMethodInterceptor.java:266)
	at org.springframework.security.authorization.method.AuthorizationManagerBeforeMethodInterceptor.invoke(AuthorizationManagerBeforeMethodInterceptor.java:197)
	at org.springframework.security.config.annotation.method.configuration.DeferringMethodInterceptor.invoke(DeferringMethodInterceptor.java:44)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:719)
	at org.frankframework.management.bus.endpoints.TestPipeline$$SpringCGLIB$$0.runTestPipeline(&lt;generated&gt;)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.messaging.handler.invocation.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:168)
	at org.springframework.integration.handler.support.IntegrationInvocableHandlerMethod.doInvoke(IntegrationInvocableHandlerMethod.java:47)
	at org.springframework.messaging.handler.invocation.InvocableHandlerMethod.invoke(InvocableHandlerMethod.java:119)
	at org.springframework.integration.handler.support.MessagingMethodInvokerHelper$HandlerMethod.invoke(MessagingMethodInvokerHelper.java:1111)
	at org.springframework.integration.handler.support.MessagingMethodInvokerHelper.invokeHandlerMethod(MessagingMethodInvokerHelper.java:576)
	at org.springframework.integration.handler.support.MessagingMethodInvokerHelper.processInternal(MessagingMethodInvokerHelper.java:489)
	at org.springframework.integration.handler.support.MessagingMethodInvokerHelper.process(MessagingMethodInvokerHelper.java:364)
	at org.springframework.integration.handler.MethodInvokingMessageProcessor.processMessage(MethodInvokingMessageProcessor.java:116)
	at org.springframework.integration.handler.ServiceActivatingHandler.handleRequestMessage(ServiceActivatingHandler.java:97)
	at org.springframework.integration.handler.AbstractReplyProducingMessageHandler.handleMessageInternal(AbstractReplyProducingMessageHandler.java:146)
	at org.springframework.integration.handler.AbstractMessageHandler.doHandleMessage(AbstractMessageHandler.java:102)
	at org.springframework.integration.handler.AbstractMessageHandler.handleMessage(AbstractMessageHandler.java:70)
	at org.springframework.integration.handler.MessageHandlerChain.lambda$configureChain$0(MessageHandlerChain.java:133)
	at org.springframework.messaging.core.GenericMessagingTemplate.doSend(GenericMessagingTemplate.java:187)
	at org.springframework.messaging.core.GenericMessagingTemplate.doSend(GenericMessagingTemplate.java:166)
	at org.springframework.messaging.core.GenericMessagingTemplate.doSend(GenericMessagingTemplate.java:47)
	at org.springframework.messaging.core.AbstractMessageSendingTemplate.send(AbstractMessageSendingTemplate.java:108)
	at org.springframework.integration.handler.AbstractMessageProducingHandler.sendOutput(AbstractMessageProducingHandler.java:556)
	at org.springframework.integration.handler.AbstractMessageProducingHandler.doProduceOutput(AbstractMessageProducingHandler.java:370)
	at org.springframework.integration.handler.AbstractMessageProducingHandler.produceOutput(AbstractMessageProducingHandler.java:289)
	at org.springframework.integration.handler.AbstractMessageProducingHandler.sendOutputs(AbstractMessageProducingHandler.java:253)
	at org.springframework.integration.handler.AbstractReplyProducingMessageHandler.handleMessageInternal(AbstractReplyProducingMessageHandler.java:152)
	at org.springframework.integration.handler.AbstractMessageHandler.doHandleMessage(AbstractMessageHandler.java:102)
	at org.springframework.integration.handler.AbstractMessageHandler.handleMessage(AbstractMessageHandler.java:70)
	at org.springframework.integration.handler.MessageHandlerChain.handleMessageInternal(MessageHandlerChain.java:161)
	at org.springframework.integration.handler.AbstractMessageHandler.doHandleMessage(AbstractMessageHandler.java:102)
	at org.springframework.integration.handler.AbstractMessageHandler.handleMessage(AbstractMessageHandler.java:70)
	at org.springframework.integration.dispatcher.BroadcastingDispatcher.invokeHandler(BroadcastingDispatcher.java:239)
	at org.springframework.integration.dispatcher.BroadcastingDispatcher.dispatch(BroadcastingDispatcher.java:196)
	at org.springframework.integration.channel.AbstractSubscribableChannel.doSend(AbstractSubscribableChannel.java:72)
	at org.springframework.integration.channel.AbstractMessageChannel.sendInternal(AbstractMessageChannel.java:438)
	at org.springframework.integration.channel.AbstractMessageChannel.send(AbstractMessageChannel.java:341)
	at org.springframework.messaging.core.GenericMessagingTemplate.doSend(GenericMessagingTemplate.java:187)
	at org.springframework.messaging.core.GenericMessagingTemplate.doSendAndReceive(GenericMessagingTemplate.java:231)
	at org.springframework.messaging.core.GenericMessagingTemplate.doSendAndReceive(GenericMessagingTemplate.java:47)
	at org.springframework.messaging.core.AbstractMessagingTemplate.sendAndReceive(AbstractMessagingTemplate.java:47)
	at org.springframework.integration.core.MessagingTemplate.sendAndReceive(MessagingTemplate.java:111)
	at org.springframework.integration.gateway.MessagingGatewaySupport.doSendAndReceive(MessagingGatewaySupport.java:681)
	at org.springframework.integration.gateway.MessagingGatewaySupport.sendAndReceive(MessagingGatewaySupport.java:609)
	at org.springframework.integration.gateway.MessagingGatewaySupport.sendAndReceiveMessage(MessagingGatewaySupport.java:583)
	at org.frankframework.management.bus.LocalGateway.sendSyncMessage(LocalGateway.java:50)
	at org.frankframework.console.controllers.FrankApiService.sendSyncMessage(FrankApiService.java:80)
	at org.frankframework.console.controllers.TestPipeline.testPipeLine(TestPipeline.java:104)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:158)
	at org.springframework.security.authorization.method.AuthorizationManagerBeforeMethodInterceptor.proceed(AuthorizationManagerBeforeMethodInterceptor.java:271)
	at org.springframework.security.authorization.method.AuthorizationManagerBeforeMethodInterceptor.attemptAuthorization(AuthorizationManagerBeforeMethodInterceptor.java:266)
	at org.springframework.security.authorization.method.AuthorizationManagerBeforeMethodInterceptor.invoke(AuthorizationManagerBeforeMethodInterceptor.java:197)
	at org.springframework.security.config.annotation.method.configuration.DeferringMethodInterceptor.invoke(DeferringMethodInterceptor.java:44)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:719)
	at org.frankframework.console.controllers.TestPipeline$$SpringCGLIB$$0.testPipeLine(&lt;generated&gt;)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.web.method.support.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:252)
	at org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest(InvocableHandlerMethod.java:184)
	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:117)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:934)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:853)
	at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:86)
	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:963)
	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:866)
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1000)
	at org.springframework.web.servlet.FrameworkServlet.doPost(FrameworkServlet.java:903)
	at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:590)
	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:874)
	at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:658)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:138)
	at org.frankframework.console.filters.SecurityLogFilter.doFilterInternal(SecurityLogFilter.java:46)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:138)
	at org.frankframework.console.filters.CsrfCookieFilter.doFilterInternal(CsrfCookieFilter.java:48)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:138)
	at org.springframework.web.filter.ShallowEtagHeaderFilter.doFilterInternal(ShallowEtagHeaderFilter.java:107)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:138)
	at org.frankframework.console.filters.CacheControlFilter.doFilter(CacheControlFilter.java:52)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:138)
	at org.frankframework.console.filters.CorsFilter.doFilter(CorsFilter.java:114)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:138)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:51)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:138)
	at org.springframework.web.filter.CompositeFilter$VirtualFilterChain.doFilter(CompositeFilter.java:108)
	at org.springframework.security.web.FilterChainProxy.lambda$doFilterInternal$3(FilterChainProxy.java:235)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:376)
	at org.springframework.security.web.access.intercept.AuthorizationFilter.doFilter(AuthorizationFilter.java:101)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:126)
	at org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:120)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.session.SessionManagementFilter.doFilter(SessionManagementFilter.java:92)
	at org.springframework.security.web.session.SessionManagementFilter.doFilter(SessionManagementFilter.java:86)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.authentication.AnonymousAuthenticationFilter.doFilter(AnonymousAuthenticationFilter.java:100)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter.doFilter(SecurityContextHolderAwareRequestFilter.java:181)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.savedrequest.RequestCacheAwareFilter.doFilter(RequestCacheAwareFilter.java:63)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:110)
	at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:96)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:110)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.context.SecurityContextHolderFilter.doFilter(SecurityContextHolderFilter.java:75)
	at org.springframework.security.web.context.SecurityContextHolderFilter.doFilter(SecurityContextHolderFilter.java:69)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:101)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:110)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:110)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:237)
	at org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:190)
	at org.springframework.web.filter.CompositeFilter$VirtualFilterChain.doFilter(CompositeFilter.java:113)
	at org.springframework.web.filter.ServletRequestPathFilter.doFilter(ServletRequestPathFilter.java:52)
	at org.springframework.web.filter.CompositeFilter$VirtualFilterChain.doFilter(CompositeFilter.java:113)
	at org.springframework.web.filter.CompositeFilter.doFilter(CompositeFilter.java:74)
	at org.springframework.security.config.annotation.web.configuration.WebSecurityConfiguration$CompositeFilterChainProxy.doFilter(WebSecurityConfiguration.java:317)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:138)
	at org.springframework.web.filter.CompositeFilter$VirtualFilterChain.doFilter(CompositeFilter.java:108)
	at org.springframework.security.web.FilterChainProxy.lambda$doFilterInternal$3(FilterChainProxy.java:235)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:376)
	at org.springframework.security.web.access.intercept.AuthorizationFilter.doFilter(AuthorizationFilter.java:101)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:126)
	at org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:120)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.session.SessionManagementFilter.doFilter(SessionManagementFilter.java:132)
	at org.springframework.security.web.session.SessionManagementFilter.doFilter(SessionManagementFilter.java:86)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.authentication.AnonymousAuthenticationFilter.doFilter(AnonymousAuthenticationFilter.java:100)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter.doFilter(SecurityContextHolderAwareRequestFilter.java:181)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.savedrequest.RequestCacheAwareFilter.doFilter(RequestCacheAwareFilter.java:63)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:110)
	at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:96)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.header.HeaderWriterFilter.doHeadersAfter(HeaderWriterFilter.java:90)
	at org.springframework.security.web.header.HeaderWriterFilter.doFilterInternal(HeaderWriterFilter.java:75)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.context.SecurityContextHolderFilter.doFilter(SecurityContextHolderFilter.java:82)
	at org.springframework.security.web.context.SecurityContextHolderFilter.doFilter(SecurityContextHolderFilter.java:69)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:101)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter.doFilterInternal(WebAsyncManagerIntegrationFilter.java:62)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.session.DisableEncodeUrlFilter.doFilterInternal(DisableEncodeUrlFilter.java:42)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116)
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:385)
	at org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:237)
	at org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:195)
	at org.springframework.web.filter.CompositeFilter$VirtualFilterChain.doFilter(CompositeFilter.java:113)
	at org.springframework.web.filter.ServletRequestPathFilter.doFilter(ServletRequestPathFilter.java:52)
	at org.springframework.web.filter.CompositeFilter$VirtualFilterChain.doFilter(CompositeFilter.java:113)
	at org.springframework.web.filter.CompositeFilter.doFilter(CompositeFilter.java:74)
	at org.springframework.security.config.annotation.web.configuration.WebSecurityConfiguration$CompositeFilterChainProxy.doFilter(WebSecurityConfiguration.java:317)
	at org.springframework.web.filter.DelegatingFilterProxy.invokeDelegate(DelegatingFilterProxy.java:355)
	at org.springframework.web.filter.DelegatingFilterProxy.doFilter(DelegatingFilterProxy.java:272)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:138)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:165)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:88)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:609)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:113)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:83)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:782)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:72)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:342)
	at org.apache.coyote.http2.StreamProcessor.service(StreamProcessor.java:476)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:63)
	at org.apache.coyote.http2.StreamProcessor.process(StreamProcessor.java:102)
	at org.apache.coyote.http2.StreamRunnable.run(StreamRunnable.java:35)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:973)
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:491)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63)
	at java.base/java.lang.Thread.run(Thread.java:1474)
Caused by: org.frankframework.core.PipeRunException: Pipe [Insert into table daggegevens] caught exception: (SaxException) Could not process list of elements using xpath [/csv/record]: (TransformerException): (SaxException): exception calling JavaListener [Store record]: Pipe [Store in database] caught exception: got exception sending message: Parameter [STN] could not parse result [Message[8887d5b:String]: [  235]] to integer: (NumberFormatException) For input string: "  235"
	at org.frankframework.pipes.MessageSendingPipe.doPipe(MessageSendingPipe.java:375)
	at org.frankframework.processors.CorePipeProcessor.processPipe(CorePipeProcessor.java:36)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.TrackPreviousPipeInMetadataProcessor.processPipe(TrackPreviousPipeInMetadataProcessor.java:42)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.CheckMessageSizePipeProcessor.processPipe(CheckMessageSizePipeProcessor.java:38)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.LockerPipeProcessor.processPipe(LockerPipeProcessor.java:40)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.TransactionAttributePipeProcessor.executePipeProcess(TransactionAttributePipeProcessor.java:96)
	at org.frankframework.processors.TransactionAttributePipeProcessor.processPipe(TransactionAttributePipeProcessor.java:66)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.LimitingParallelExecutionPipeProcessor.processPipe(LimitingParallelExecutionPipeProcessor.java:50)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.LimitingParallelExecutionPipeProcessor.processPipe(LimitingParallelExecutionPipeProcessor.java:77)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:158)
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:94)
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugPipeGetInputFrom(IbisDebuggerAdvice.java:187)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:648)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:630)
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:70)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:719)
	at org.frankframework.processors.LimitingParallelExecutionPipeProcessor$$SpringCGLIB$$0.processPipe(&lt;generated&gt;)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.ExceptionHandlingPipeProcessor.processPipe(ExceptionHandlingPipeProcessor.java:40)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.InputOutputPipeProcessor.processPipe(InputOutputPipeProcessor.java:97)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.InputOutputPipeProcessor.processPipe(InputOutputPipeProcessor.java:240)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:158)
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:94)
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugPipeInputOutputAbort(IbisDebuggerAdvice.java:159)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:648)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:630)
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:70)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:719)
	at org.frankframework.processors.InputOutputPipeProcessor$$SpringCGLIB$$0.processPipe(&lt;generated&gt;)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.MonitoringPipeProcessor.processPipe(MonitoringPipeProcessor.java:40)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.LogPipeProcessor.processPipe(LogPipeProcessor.java:56)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.CorePipeLineProcessor.runToExit(CorePipeLineProcessor.java:222)
	at org.frankframework.processors.CorePipeLineProcessor.processPipeLine(CorePipeLineProcessor.java:64)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:158)
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:94)
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugPipeLineInputOutputAbort(IbisDebuggerAdvice.java:124)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:648)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:630)
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:70)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:719)
	at org.frankframework.processors.CorePipeLineProcessor$$SpringCGLIB$$0.processPipeLine(&lt;generated&gt;)
	at org.frankframework.processors.LockerPipeLineProcessor.processPipeLine(LockerPipeLineProcessor.java:50)
	at org.frankframework.processors.TransactionAttributePipeLineProcessor.processPipeLine(TransactionAttributePipeLineProcessor.java:51)
	at org.frankframework.processors.LimitingParallelExecutionPipeLineProcessor.processPipeLine(LimitingParallelExecutionPipeLineProcessor.java:45)
	at org.frankframework.processors.CachePipeLineProcessor.processPipeLine(CachePipeLineProcessor.java:43)
	at org.frankframework.processors.InputOutputPipeLineProcessor.processPipeLine(InputOutputPipeLineProcessor.java:48)
	at org.frankframework.core.PipeLine.process(PipeLine.java:441)
	at org.frankframework.core.Adapter.processMessageWithExceptions(Adapter.java:840)
	... 217 more
Caused by: org.frankframework.core.SenderException: (SaxException) Could not process list of elements using xpath [/csv/record]: (TransformerException): (SaxException): exception calling JavaListener [Store record]: Pipe [Store in database] caught exception: got exception sending message: Parameter [STN] could not parse result [Message[8887d5b:String]: [  235]] to integer: (NumberFormatException) For input string: "  235"
	at org.frankframework.pipes.ForEachChildElementPipe.iterateOverInput(ForEachChildElementPipe.java:402)
	at org.frankframework.pipes.IteratingPipe.sendMessage(IteratingPipe.java:477)
	at org.frankframework.pipes.MessageSendingPipe.sendMessageWithRetries(MessageSendingPipe.java:414)
	at org.frankframework.pipes.MessageSendingPipe.doPipe(MessageSendingPipe.java:347)
	... 308 more
Caused by: org.frankframework.xml.SaxException: Could not process list of elements using xpath [/csv/record]
	at org.frankframework.pipes.ForEachChildElementPipe$1.handleException(ForEachChildElementPipe.java:353)
	at org.frankframework.xml.AbstractExceptionCatchingFilter.characters(AbstractExceptionCatchingFilter.java:65)
	at org.apache.xerces.parsers.AbstractSAXParser.characters(Unknown Source)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanContent(Unknown Source)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl$FragmentContentDispatcher.dispatch(Unknown Source)
	at org.apache.xerces.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source)
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source)
	at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source)
	at org.apache.xerces.parsers.XMLParser.parse(Unknown Source)
	at org.apache.xerces.parsers.AbstractSAXParser.parse(Unknown Source)
	at org.apache.xerces.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source)
	at org.frankframework.util.XmlUtils.parseXml(XmlUtils.java:259)
	at org.frankframework.util.XmlUtils.parseXml(XmlUtils.java:246)
	at org.frankframework.pipes.ForEachChildElementPipe.iterateOverInput(ForEachChildElementPipe.java:392)
	... 311 more
Caused by: javax.xml.transform.TransformerException: org.frankframework.xml.SaxException: exception calling JavaListener [Store record]: Pipe [Store in database] caught exception: got exception sending message: Parameter [STN] could not parse result [Message[8887d5b:String]: [  235]] to integer: (NumberFormatException) For input string: "  235"
	at org.apache.xalan.templates.ElemCopyOf.execute(ElemCopyOf.java:212)
	at org.apache.xalan.transformer.TransformerImpl.executeChildTemplates(TransformerImpl.java:2402)
	at org.apache.xalan.templates.ElemElement.constructNode(ElemElement.java:341)
	at org.apache.xalan.templates.ElemElement.execute(ElemElement.java:290)
	at org.apache.xalan.transformer.TransformerImpl.executeChildTemplates(TransformerImpl.java:2402)
	at org.apache.xalan.transformer.TransformerImpl.applyTemplateToNode(TransformerImpl.java:2272)
	at org.apache.xalan.transformer.TransformerImpl.transformNode(TransformerImpl.java:1358)
	at org.apache.xalan.transformer.TransformerImpl.run(TransformerImpl.java:3449)
	... 1 more
Caused by: org.frankframework.xml.SaxException: exception calling JavaListener [Store record]: Pipe [Store in database] caught exception: got exception sending message: Parameter [STN] could not parse result [Message[8887d5b:String]: [  235]] to integer: (NumberFormatException) For input string: "  235"
	at org.frankframework.pipes.ForEachChildElementPipe$ItemCallbackCallingHandler.endNode(ForEachChildElementPipe.java:212)
	at org.frankframework.xml.NodeSetFilter.endElement(NodeSetFilter.java:163)
	at org.frankframework.pipes.ForEachChildElementPipe$ItemCallbackCallingHandler.endElement(ForEachChildElementPipe.java:231)
	at java.xml/org.xml.sax.helpers.XMLFilterImpl.endElement(XMLFilterImpl.java:558)
	at org.frankframework.documentbuilder.xml.XmlTee.endElement(XmlTee.java:64)
	at java.xml/org.xml.sax.helpers.XMLFilterImpl.endElement(XMLFilterImpl.java:558)
	at org.frankframework.xml.NamespaceRemovingFilter.endElement(NamespaceRemovingFilter.java:35)
	at java.xml/org.xml.sax.helpers.XMLFilterImpl.endElement(XMLFilterImpl.java:558)
	at org.frankframework.xml.AbstractExceptionCatchingFilter.endElement(AbstractExceptionCatchingFilter.java:72)
	at org.apache.xml.serializer.ToXMLSAXHandler.endElement(ToXMLSAXHandler.java:263)
	at org.apache.xml.dtm.ref.DTMTreeWalker.endNode(DTMTreeWalker.java:375)
	at org.apache.xalan.transformer.TreeWalker2Result.endNode(TreeWalker2Result.java:87)
	at org.apache.xml.dtm.ref.DTMTreeWalker.traverse(DTMTreeWalker.java:134)
	at org.apache.xalan.transformer.TreeWalker2Result.traverse(TreeWalker2Result.java:74)
	at org.apache.xalan.templates.ElemCopyOf.execute(ElemCopyOf.java:186)
	... 8 more
Caused by: org.frankframework.core.SenderException: exception calling JavaListener [Store record]: Pipe [Store in database] caught exception: got exception sending message: Parameter [STN] could not parse result [Message[8887d5b:String]: [  235]] to integer: (NumberFormatException) For input string: "  235"
	at org.frankframework.senders.IbisLocalSender.sendMessage(IbisLocalSender.java:284)
	at org.frankframework.core.ISender.sendMessageOrThrow(ISender.java:72)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:158)
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:94)
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugSenderInputOutputAbort(IbisDebuggerAdvice.java:231)
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugSenderSendMessageOrThrow(IbisDebuggerAdvice.java:284)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:648)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:630)
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:70)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:719)
	at org.frankframework.senders.IbisLocalSender$$SpringCGLIB$$0.sendMessageOrThrow(&lt;generated&gt;)
	at org.frankframework.pipes.IteratingPipe$ItemCallback.handleItem(IteratingPipe.java:329)
	at org.frankframework.pipes.StringIteratorPipe$1.handleItem(StringIteratorPipe.java:126)
	at org.frankframework.pipes.StringIteratorPipe$1.handleItem(StringIteratorPipe.java:68)
	at org.frankframework.pipes.ForEachChildElementPipe$ItemCallbackCallingHandler.endNode(ForEachChildElementPipe.java:208)
	... 22 more
Caused by: org.frankframework.core.ListenerException: Pipe [Store in database] caught exception: got exception sending message: Parameter [STN] could not parse result [Message[8887d5b:String]: [  235]] to integer: (NumberFormatException) For input string: "  235"
	at org.frankframework.core.Adapter.processMessageWithExceptions(Adapter.java:843)
	at org.frankframework.receivers.Receiver.processMessageInAdapter(Receiver.java:1271)
	at org.frankframework.receivers.Receiver.processRequest(Receiver.java:1047)
	at org.frankframework.receivers.JavaListener.processRequest(JavaListener.java:181)
	at org.frankframework.receivers.JavaListener.processRequest(JavaListener.java:170)
	at org.frankframework.senders.IbisLocalSender.sendMessage(IbisLocalSender.java:277)
	... 45 more
Caused by: org.frankframework.core.PipeRunException: Pipe [Store in database] caught exception: got exception sending message: Parameter [STN] could not parse result [Message[8887d5b:String]: [  235]] to integer: (NumberFormatException) For input string: "  235"
	at org.frankframework.pipes.MessageSendingPipe.doPipe(MessageSendingPipe.java:375)
	at org.frankframework.processors.CorePipeProcessor.processPipe(CorePipeProcessor.java:36)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.TrackPreviousPipeInMetadataProcessor.processPipe(TrackPreviousPipeInMetadataProcessor.java:42)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.CheckMessageSizePipeProcessor.processPipe(CheckMessageSizePipeProcessor.java:38)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.LockerPipeProcessor.processPipe(LockerPipeProcessor.java:40)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.TransactionAttributePipeProcessor.executePipeProcess(TransactionAttributePipeProcessor.java:96)
	at org.frankframework.processors.TransactionAttributePipeProcessor.processPipe(TransactionAttributePipeProcessor.java:66)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.LimitingParallelExecutionPipeProcessor.processPipe(LimitingParallelExecutionPipeProcessor.java:50)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.LimitingParallelExecutionPipeProcessor.processPipe(LimitingParallelExecutionPipeProcessor.java:77)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:158)
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:94)
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugPipeGetInputFrom(IbisDebuggerAdvice.java:187)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:648)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:630)
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:70)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:719)
	at org.frankframework.processors.LimitingParallelExecutionPipeProcessor$$SpringCGLIB$$0.processPipe(&lt;generated&gt;)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.ExceptionHandlingPipeProcessor.processPipe(ExceptionHandlingPipeProcessor.java:40)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.InputOutputPipeProcessor.processPipe(InputOutputPipeProcessor.java:97)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.InputOutputPipeProcessor.processPipe(InputOutputPipeProcessor.java:240)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:158)
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:94)
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugPipeInputOutputAbort(IbisDebuggerAdvice.java:159)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:648)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:630)
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:70)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:719)
	at org.frankframework.processors.InputOutputPipeProcessor$$SpringCGLIB$$0.processPipe(&lt;generated&gt;)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.MonitoringPipeProcessor.processPipe(MonitoringPipeProcessor.java:40)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.LogPipeProcessor.processPipe(LogPipeProcessor.java:56)
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:47)
	at org.frankframework.processors.CorePipeLineProcessor.runToExit(CorePipeLineProcessor.java:222)
	at org.frankframework.processors.CorePipeLineProcessor.processPipeLine(CorePipeLineProcessor.java:64)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:158)
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:94)
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugPipeLineInputOutputAbort(IbisDebuggerAdvice.java:124)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:648)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:630)
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:70)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:719)
	at org.frankframework.processors.CorePipeLineProcessor$$SpringCGLIB$$0.processPipeLine(&lt;generated&gt;)
	at org.frankframework.processors.LockerPipeLineProcessor.processPipeLine(LockerPipeLineProcessor.java:62)
	at org.frankframework.processors.TransactionAttributePipeLineProcessor.processPipeLine(TransactionAttributePipeLineProcessor.java:51)
	at org.frankframework.processors.LimitingParallelExecutionPipeLineProcessor.processPipeLine(LimitingParallelExecutionPipeLineProcessor.java:45)
	at org.frankframework.processors.CachePipeLineProcessor.processPipeLine(CachePipeLineProcessor.java:43)
	at org.frankframework.processors.InputOutputPipeLineProcessor.processPipeLine(InputOutputPipeLineProcessor.java:48)
	at org.frankframework.core.PipeLine.process(PipeLine.java:441)
	at org.frankframework.core.Adapter.processMessageWithExceptions(Adapter.java:840)
	... 50 more
Caused by: org.frankframework.core.SenderException: got exception sending message: Parameter [STN] could not parse result [Message[8887d5b:String]: [  235]] to integer: (NumberFormatException) For input string: "  235"
	at org.frankframework.jdbc.AbstractJdbcQuerySender.executeStatementSet(AbstractJdbcQuerySender.java:356)
	at org.frankframework.jdbc.FixedQuerySender.sendMessage(FixedQuerySender.java:131)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:158)
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:94)
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugSenderInputOutputAbort(IbisDebuggerAdvice.java:231)
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugBlockEnabledSenderInputOutputAbort(IbisDebuggerAdvice.java:292)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:648)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:630)
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:70)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:719)
	at org.frankframework.jdbc.FixedQuerySender$$SpringCGLIB$$0.sendMessage(&lt;generated&gt;)
	at org.frankframework.jdbc.AbstractJdbcSender.sendMessage(AbstractJdbcSender.java:132)
	at org.frankframework.pipes.MessageSendingPipe.sendMessage(MessageSendingPipe.java:621)
	at org.frankframework.pipes.MessageSendingPipe.sendMessageWithRetries(MessageSendingPipe.java:414)
	at org.frankframework.pipes.MessageSendingPipe.doPipe(MessageSendingPipe.java:347)
	... 141 more
Caused by: org.frankframework.core.ParameterException: Parameter [STN] could not parse result [Message[8887d5b:String]: [  235]] to integer: (NumberFormatException) For input string: "  235"
	at org.frankframework.parameters.NumberParameter.getValueAsType(NumberParameter.java:140)
	at org.frankframework.parameters.NumberParameter.getValueAsType(NumberParameter.java:37)
	at org.frankframework.parameters.AbstractParameter.getValue(AbstractParameter.java:492)
	at org.frankframework.parameters.NumberParameter.getValue(NumberParameter.java:95)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:158)
	at org.springframework.aop.aspectj.MethodInvocationProceedingJoinPoint.proceed(MethodInvocationProceedingJoinPoint.java:82)
	at org.frankframework.ladybug.IbisDebuggerAdvice.debugParameterResolvedTo(IbisDebuggerAdvice.java:474)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:648)
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:630)
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:70)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:168)
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:719)
	at org.frankframework.parameters.NumberParameter$$SpringCGLIB$$0.getValue(&lt;generated&gt;)
	at org.frankframework.parameters.ParameterList.getValue(ParameterList.java:207)
	at org.frankframework.parameters.ParameterList.getValues(ParameterList.java:178)
	at org.frankframework.parameters.ParameterList.getValues(ParameterList.java:160)
	at org.frankframework.util.JdbcUtil.applyParameters(JdbcUtil.java:474)
	at org.frankframework.jdbc.AbstractJdbcQuerySender.executeStatementSet(AbstractJdbcQuerySender.java:321)
	... 164 more
Caused by: java.lang.NumberFormatException: For input string: "  235"
	at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)
	at java.base/java.lang.Integer.parseInt(Integer.java:565)
	at java.base/java.lang.Integer.parseInt(Integer.java:662)
	at org.frankframework.parameters.NumberParameter.getValueAsType(NumberParameter.java:138)
	... 189 more
</details>
	<originalMessage messageId="testmessage-0a000066-15379153_19eb129e969_-7fb0" receivedTime="Wed Jun 10 12:58:08 CEST 2026" nil="true"/>
</errorMessage>
```

--- a/core/src/main/java/org/frankframework/parameters/DateParameter.java
+++ b/core/src/main/java/org/frankframework/parameters/DateParameter.java
@@ -137,7 +137,7 @@ public class DateParameter extends AbstractParameter<Date> {
 			String sanitized = value.replace(".", "")
 									.replace(",", "")
 									.replace("_", "");
-			long epoch = Long.parseLong(sanitized);
+			long epoch = Long.parseLong(sanitized.trim());
 
 			if (epoch < 1_000_000_000_000L) {
 				epoch *= 1000L; // convert seconds to milliseconds

--- a/core/src/main/java/org/frankframework/parameters/NumberParameter.java
+++ b/core/src/main/java/org/frankframework/parameters/NumberParameter.java
@@ -21,6 +21,7 @@ import java.io.IOException;
 import java.text.DecimalFormat;
 import java.text.DecimalFormatSymbols;
 import java.text.ParseException;
+import java.util.Objects;
 
 import org.apache.commons.lang3.StringUtils;
 import org.jspecify.annotations.NonNull;
@@ -130,12 +131,12 @@ public class NumberParameter extends AbstractParameter<Number> {
 			}
 		}
 		if(getType() == ParameterType.INTEGER) {
-			if (request.asObject() instanceof Integer integer) {
-				return integer;
+			if (request.asObject() instanceof Number number) {
+				return number;
 			}
 			log.debug("Parameter [{}] converting result [{}] to integer", this::getName, ()->request);
 			try {
-				return Integer.parseInt(request.asString());
+				return Integer.parseInt(Objects.requireNonNull(request.asString()).trim());
 			} catch (NumberFormatException e) {
 				throw new ParameterException(getName(), "Parameter [" + getName() + "] could not parse result [" + request + "] to integer", e);
 			}

--- a/core/src/main/java/org/frankframework/parameters/ParameterValue.java
+++ b/core/src/main/java/org/frankframework/parameters/ParameterValue.java
@@ -105,7 +105,7 @@ public class ParameterValue {
 			return i;
 		}
 
-		return Integer.parseInt(valueAsString());
+		return Integer.parseInt(valueAsString().trim());
 	}
 
 	/**
@@ -113,7 +113,7 @@ public class ParameterValue {
 	 * @return convert the value to a long
 	 */
 	public long asLongValue(long defaultValue) {
-		return value != null ? Long.parseLong(valueAsString()) : defaultValue;
+		return value != null ? Long.parseLong(valueAsString().trim()) : defaultValue;
 	}
 
 	/**

--- a/core/src/test/java/org/frankframework/parameters/NumberParameterTest.java
+++ b/core/src/test/java/org/frankframework/parameters/NumberParameterTest.java
@@ -17,6 +17,22 @@ import org.frankframework.util.TimeProvider;
 public class NumberParameterTest {
 
 	@Test
+	public void testNumberWithSpacesInValue() throws Exception {
+		NumberParameter p = new NumberParameter();
+		p.setName("number");
+		p.setValue(" 8 ");
+		p.configure();
+
+		Object result = p.getValue(new Message("fakeMessage"), null).getValue();
+		Integer i1 = assertInstanceOf(Integer.class, result);
+		assertEquals(8, i1.intValue());
+
+		Number valueAsType = p.getValueAsType(new Message(" 9 "), false);
+		Integer i2 = assertInstanceOf(Integer.class, valueAsType);
+		assertEquals(9, i2.intValue());
+	}
+
+	@Test
 	public void testNumberWithMinLength() throws Exception {
 		NumberParameter p = new NumberParameter();
 		p.setName("number");
@@ -28,7 +44,7 @@ public class NumberParameterTest {
 
 		Object result = p.getValue(message, null).getValue();
 		assertInstanceOf(String.class, result);
-		assertEquals("0000000008", (String) result);
+		assertEquals("0000000008", result);
 	}
 
 	@Test

